html/changelogs/AutoChangeLog-pr-964.yml

```diff
@@ -0,0 +1,4 @@
+author: "thegrb93"
+delete-after: True
+changes:
+  - rscadd: "Borg dominatrix module to lustwish that gives them some dominatrix toys"
```

Comment:
No idea if these changelog entries shoudl be here

Reply:
No longer in